### PR TITLE
BB-18-Event-Creation-Modal-Component 

### PR DESCRIPTION
**Pull request by LeonJia1**
## Describe your changes:
Created form in a modal. Basically has a bunch of input fields a few drop downs (i copied bruno lol) an expanding description box, and an upload picture.

## Proof that your changes work (include screenshots and/or for frontend):
Screenshot 2023-04-17 at 8 57 17 AM
Screenshot 2023-04-17 at 8 57 11 AM

## Checklist before requesting a review
- [ y] I have performed a self-review of my code
- [y ] I have DM'd the group chat that I have made a PR and need reviewers.
- [ y] I haved moved the Jira issue to "Under Review"
- [ y] I have run `npx eslint filename.js` on all JavaScript files I've changed in this PR and fixed all displayed errors
- [y ] I have checked the Chrome console in Developer Tools to make sure there are no significant errors

### CODE REVIEW

File: frontend/src/components/CreationModal/CreationModal.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                                    flip="true"
+                                    bsPrefix={styles.dropdown_button2}
+                                    onSelect={handleSelectCategory}
+                                >
+                                    <Dropdown.Item eventKey="female">Food</Dropdown.Item>
+                                    <Dropdown.Item eventKey="male">Hike</Dropdown.Item>
+                                    <Dropdown.Item eventKey="other">Games</Dropdown.Item>
+                                    <Dropdown.Item eventKey="female">Drinks</Dropdown.Item>
+                                    <Dropdown.Item eventKey="male">Virtual</Dropdown.Item>
+                                    <Dropdown.Item eventKey="other">Sports</Dropdown.Item>
```
Comment (DylanHopkins21) [inline frontend/src/components/CreationModal/CreationModal.js:78]: What are all these eventKey=male, female, other? Idk what it is so I'm not gonna ask you to change it, I trust it is functioning fine

**Review by DylanHopkins21 — APPROVED** [review]
Made one comment, just wanted you to take a look and make sure you weren't just blindly copying from Bruno or something, I don't know exactly what those key things do. Other than that it looks perfect

File: frontend/src/components/CreationModal/CreationModal.js
Comment (thomaspwang) [inline frontend/src/components/CreationModal/CreationModal.js:1]: textfield titles should be bolded! (e.g "Date", "Name", "Upload Image", etc)

File: frontend/src/components/CreationModal/CreationModal.js (hunk 1 above)
Comment (thomaspwang) [reply frontend/src/components/CreationModal/CreationModal.js:78]: i think they're categories he came up with

you're going to have to change it when you implement categories but its fine for now

**Review by thomaspwang — CHANGES_REQUESTED** [review]
LGTM! Just one change (bold the titles), and we're good to go.

File: frontend/src/components/CreationModal/CreationModal.js (hunk 1 above)
Comment (DylanHopkins21) [reply frontend/src/components/CreationModal/CreationModal.js:78]: yeah sorry i meant specifically why does Hike have eventKey="male" and Food has eventKey="female"

Comment (LeonJia1) [reply frontend/src/components/CreationModal/CreationModal.js:78]: ah my bad i didnt change the event keys only the text. fixed

File: frontend/src/components/CreationModal/CreationModal.js
Comment (LeonJia1) [reply frontend/src/components/CreationModal/CreationModal.js:1]: done

**Comment by LeonJia1** [comment]
Screenshot 2023-04-24 at 10 26 24 AM

fixed vers

**Review by thomaspwang — APPROVED** [review]
LGTM!

**Pull request merged**